### ISSUE TITLE
Arrow functions & UglifyJS

### ISSUE THREAD

**Issue by MitchTalmadge** [jprichardson/node-fs-extra#418]
I'm probably doing something wrong, so I'm looking for advice. I am creating an Electron project that uses `fs-extra`, and I use Webpack to minify the project, with UglifyJS. In my `tsconfig.json`, I set the `target` to `es5` because UglifyJS cannot parse ES6, from what I've read. 

Everything was packaging fine until I added `fs-extra`; other packages, like `angular`, `moment`, `electron`, etc. have worked fine, but `fs-extra` will not compile due to its use of arrow functions. Here is the error:

```
ERROR in ./resources/scripts/main-938b3a8df63c9ba3e45b.js from UglifyJs
Unexpected token: operator (>) [./resources/scripts/main-938b3a8df63c9ba3e45b.js:21117,29]
```

Caused by this:

```javascript
// Export all keys:
Object.keys(fs).forEach(key => {
  exports[key] = fs[key]
})
```

Which can be found [here](https://github.com/jprichardson/node-fs-extra/blob/master/lib/fs/index.js#L42).

What I've noticed is that none of my other dependencies use arrow functions, or other ES6 features. They all seem to be ES5 or lower. Have I just lucked out until now? I don't know where to go from here, but I thought I'd bring this up, hoping that someone can give me some pointers. Thanks

**Comment by MitchTalmadge** [comment]
Using [babel-loader](https://github.com/babel/babel-loader) and not excluding `node_modules` solved this problem, though it takes forever to build. Maybe `fs-extra` should be transpiled to ES5 before published to npm, at least until UglifyJS is caught up to ES6. I can't imagine I'm the only one with this problem, but I've been wrong before.

**Comment by manidlou** [comment]
@MitchTalmadge I am glad you solved our issue. However, as it is specified in [changelog](https://github.com/jprichardson/node-fs-extra/blob/master/CHANGELOG.md#changed-1) ES6 features including arrow functions were added to `fs-extra` codebase in version `2.1.0`. If you want, as you probably know, you can install specific version like `npm i fs-extra@2.0.0`. You just need to make sure that the `fs-extra` features that you are using in your project are available in `v2.0.0`.

**Comment by MitchTalmadge** [comment]
I see. Hopefully UglifyJS catches up soon. Until then, this works for me. Thanks!
